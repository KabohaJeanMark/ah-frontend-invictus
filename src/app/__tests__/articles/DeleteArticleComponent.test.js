import React from 'react';
import { shallow } from "enzyme";
import DeleteArticleComponent from "../../components/DeleteArticle";
import toJson from 'enzyme-to-json';

describe('<DeleteArticleComponent/>', () => {
    let wrapper;
    const Props = {};
    it('renders delete article component', () => {
        wrapper = shallow(<DeleteArticleComponent {...Props} />);
    });
    it('component matches snapshot', () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});

