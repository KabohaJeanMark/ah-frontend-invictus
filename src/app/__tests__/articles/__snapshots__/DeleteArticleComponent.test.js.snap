// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeleteArticleComponent/> component matches snapshot 1`] = `
<div>
  <button
    className="btn btn-outline-danger"
  >
    Delete
  </button>
</div>
`;
